### ISSUE TITLE
unable to compile kernel driver for kernel 5.4.1

### ISSUE THREAD

**Issue by gianlucapettinello** [DIGImend/digimend-kernel-drivers#326]
After upgrading kernel to 5.4.1 I get an error when dkms compiles digimend kernel driver. See attached make.log for more info. OS: Archlinux
[make.log](https://github.com/DIGImend/digimend-kernel-drivers/files/3913499/make.log)

**Comment by FraYoshi** [comment]
Same for kernel 5.4.2-1 (archlinux) dkms build.

```
Creating symlink /var/lib/dkms/digimend/10/source ->
                 /usr/src/digimend-10

DKMS: add completed.
dkms build digimend/10

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
make -j16 KERNELRELEASE=5.4.2-arch1-1 KVERSION=5.4.2-arch1-1...(bad exit status: 2)
Error! Bad return status for module build on kernel: 5.4.2-arch1-1 (x86_64)
Consult /var/lib/dkms/digimend/10/build/make.log for more information.
make: *** [Makefile:74: dkms_modules_install] Error 10
```

Make Log in detail:
[make.log](https://github.com/DIGImend/digimend-kernel-drivers/files/3943718/make.log)

**Comment by smt923** [comment]
Also an issue on 5.4.3, same error as above

**Comment by gonX** [comment]
See if #331 fixes it for you, thanks

**Comment by smt923** [comment]
making the change in the PR allowed it to build for me on 5.4, thanks!

**Comment by FraYoshi** [comment]
Solved also for me in 5.4.2-arch1-1, thanks!

**Comment by spbnick** [comment]
Thanks everybody for the reports and @gonX for the fix! Merged now.
